FSI/Stress.C

```diff
@@ -0,0 +1,142 @@
+#include "Stress.H"
+
+using namespace Foam;
+
+preciceAdapter::FSI::Stress::Stress
+(
+    const Foam::fvMesh& mesh,
+    const fileName& timeName,
+    const std::string solverType
+    /* TODO: We should add any required field names here.
+    /  They would need to be vector fields.
+    /  See FSI/Temperature.C for details.
+    */
+)
+:
+Force(mesh,solverType),
+mesh_(mesh),
+solverType_(solverType)
+{
+    //What about type "basic"?
```

Comment:
What would be an example? I think the error message is enough.

Reply:
This is a copied comment from the force file, not mine. I think it can be deleted.